tests/integration/test_commands.py

```diff
@@ -665,18 +665,18 @@ def test_generate_bulk_rna_google_sheet_manifest(
             # Reset config to it's default values
             CONFIG.load_config("config_example.yml")
 
-        assert result.output.split("\n")[7] == (
+        assert result.output.split("\n")[8] == (
```

Comment:
Ah this is to account for the new data model URL column added right?

Reply:
No,  the changes in this PR change the output text because it add lines for displaying progress bars for the file view query necessary for getting the files. The test is checking specific lines of the text output for content and now that content is on a different line. This test wasn't present when I started developing this PR but was pulled in with `develop` later in the process.